### ISSUE TITLE
no-paragraph-content-indent: image as first child

### ISSUE THREAD

**Issue by wooorm** [wooorm/remark-lint#155]
Images alt text is not present in the tree through `value`, but through `alt`, thus causing `no-paragraph-content-indent` to warn.

For example:

```markdown
![alt](url) text
```

Yields:

```text
example.md
  1:11  warning  Expected no indentation in paragraph content  no-paragraph-content-indent  remark-lint

⚠ 1 warning
```

Source: https://github.com/wooorm/remark-lint/issues/153#issuecomment-318331349.

**Comment by wooorm** [comment]
@evilebottnawi Fixed and published!
